client/src/components/App/Content/LandingPage/adminLandingPage/investigationsInfo/InvestigationBarChart/InvestigationBarChart.tsx

```diff
@@ -23,6 +23,7 @@ const InvestigationBarChart: React.FC<Props> = ({ investigationsGraphData }): JS
                     <b style={{color}} className={classes.barTooltip}>{indexValue}: {value}</b>
                 )
             }}
+            theme={{tooltip: {container: {zIndex: 9999, position: 'absolute'}}}}
```

Comment:
why not writing `zIndex: 9999, position: 'absolute'` in InvestigationBarChartStyles